src/clients/feed/feed.ts

```diff
@@ -31,12 +31,6 @@ export type Status =
   | "unread"
   | "unarchived";
 
-function invertStatus(status: Status): Status {
```

Comment:
Curious why this got cut? Just don't need it anymore?

Reply:
Yeah, because we're using the batch endpoint we don't need this